modxcms/revolution
# Fix display of notification when User password is created/updated

### What does it do?
Corrects conditionals in New Password area of the User form so the new password dialog notification only shows when the password is MODX-generated.

### Why is it needed?
See related issue.

### How to test
Create and update users using each password method to verify the dialog only shows when appropriate.

### Related issue(s)/PR(s)
Resolves #16666
